tokio-util/src/sync/mod.rs

```diff
@@ -5,6 +5,9 @@ pub use cancellation_token::{
     guard::DropGuard, CancellationToken, WaitForCancellationFuture, WaitForCancellationFutureOwned,
 };
 
+pub mod task_tracker;
+pub use task_tracker::TaskTracker;
```

Comment:
This is currently in `tokio_util::sync`. Should I move it to `tokio_util::task`?

Reply:
I think either is fine. I'm slightly inclined to say task though, because it's kinda similar to JoinSet.

Reply:
Renaming it.